pkg/server/server.go

```diff
@@ -104,6 +107,29 @@ func (s Server) Shutdown() {
 	}
 }
 
+func wrapHandler(handler http.Handler, conf config.PluginConfigs) http.Handler {
```

Comment:
`wrapHandler` is not very descriptive, consider:
```suggestion
func wrapWithInterceptors(handler http.Handler, conf config.PluginConfigs) http.Handler {
```